source/jormungandr/jormungandr/street_network/street_network.py

```diff
@@ -38,16 +38,23 @@
 # http://stackoverflow.com/a/38668373/1614576
 ABC = abc.ABCMeta(str("ABC"), (object,), {})
 
+# we need to "invert" a direct path when it's a ending fallback by car if and only if it's returned by kraken
+# In other case, it's ignored
+class DirectPathType:
+    NORMAL = 0
+    ENDING_CAR_FALLBACK_BY_KRAKEN = 1
```

Comment:
We cannot afford a so connector-specific name/behaviour in generic class / experimental scenario.
Functionnally we have 3 types of directPath (cf. previous comment), then cache (and so make_direct_path_key) must be connector-specific, and in the connector we can choose to manage 2 of the 3 DirectPathType's the same way...
But it's the connector's responsability.